src/dropdown.stories.ts

```diff
@@ -129,18 +129,48 @@ const meta: Meta = {
         type: {
           summary: 'method',
           detail: `
-async (query: string): Promise<DropdownOption[] | void> => {
```

Comment:
Rounded out the existing example and added one for server-side filtering. Screenshot below.